python/fio/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/centos/centos:stream8
+
+RUN dnf -y module install python39 && dnf -y install python39 python39-pip
+RUN dnf -y install fio-3.19-3.el8
+RUN mkdir /plugin
+COPY requirements.txt /plugin
+RUN pip3 install --requirement /plugin/requirements.txt
+COPY fio_plugin.py /plugin
+COPY fio_schema.py /plugin
+
+WORKDIR /plugin
+ENTRYPOINT ["python3.9", "fio_plugin.py" ]
+CMD []
+
+LABEL org.opencontainers.image.source="https://github.com/mfleader/arcaflow-plugins/blob/fio/python/_fio_plugin/fio_plugin.py"
```

Comment:
The label should point to the upstream arcaflow repo and the fio plugin therein.

Reply:
how does one do that if the plugin does not exist in that repo yet?

Reply:
like this `https://github.com/arcalot/arcaflow-plugins/tree/main/python/fio`

Reply:
Yeah. For uperf I set it to `https://github.com/arcalot/arcaflow-plugins/blob/main/python/uperf/Dockerfile` by anticipating that that will be the URL.
I also pointed it directly to the Dockerfile, because that is how I interpreted what that's supposed to be:
- URL to get source code for building the image (string)